examples/filtered_lifting_line/with_correction/5MW_Baseline/Airfoils/Cylinder1.dat

```diff
@@ -0,0 +1,58 @@
+! ------------ AirfoilInfo v1.01.x Input File ----------------------------------
```

Comment:
Please remove duplicate 5MW input files. Use `symlink` and refer to the input deck in regression tests.